plugins/eslint-plugin-aws-toolkits/index.ts

```diff
@@ -19,6 +20,7 @@ const rules = {
     'no-string-exec-for-child-process': NoStringExecForChildProcess,
     'no-console-log': NoConsoleLog,
     'no-json-stringify-in-log': noJsonStringifyInLog,
+    'no-string-sub-mismatch': noStringSubMismatch,
```

Comment:
"printf" might be a slightly more recognizable name. Or something like that. "string substitution" sounds much broader.
```suggestion
    'no-printf-mismatch': noStringSubMismatch,
```